### DISCUSSION TITLE
Pattern contribution for Data handling

### DISCUSSION THREAD

**Discussion by sang-yeo** [twilio-labs/paste#1207] [General] [answered]
👋🏽 I started to work on the pattern contribution spec doc regarding Data handling in Twilio console and beyond. 

Here is the WIP [spec doc](https://docs.google.com/document/d/1Kttt8ot5GW39t946ZKdc1qE0_XcUNduk9DzfUE4iiuE/edit?usp=sharing) and I'd like to have a kickoff meeting with DSYS to discuss, review the spec and share findings. 

Please let me know who do I invite and when. 

Cheers.

**Comment by gloriliale** [comment, marked as answer]
Hi @sang-yeo! There's a PD working on a similar pattern, Johanna Weintraub from Console, so I'd recommend syncing up with her before we do a kickoff, to reduce overlap. Thanks!

**Reply by sang-yeo** [reply]
Hi @gloriliale, I synced with Jo. We've discovered difference between Jo's and my contribution area and discussed about how to collaborate in the later stage. 
We(me and jo)'d like to have a kickoff meeting with Dsys team and move forward. please let me know who I need to invite for the kickoff.

**Reply by gloriliale** [reply]
Hi @sang-yeo! ~~I'll be getting back to you about this tomorrow!~~ Looks like you've got it scheduled, exciting!
